### DISCUSSION TITLE
halber Jahresbeitrag

### DISCUSSION THREAD

**Discussion by tolot27** [openjverein/jverein#343] [General]
Geht das mit dem hälftigen Jahresbeitrag für unterjährig eintretende Mitglieder immer noch nur über eine gesonderte Beitragsgruppe, wie in [Halber Jahresbeitrag für Späteintreter](https://jverein-forum.de/viewtopic.php?t=3252) beschrieben?
Falls ja, müsste ich alle Beitragsgruppen duplizieren. Ein einmaliger Rabatt könnte es ja ggf. auch lösen.

**Comment by lenilsas** [comment]
Ich mache es so, dass ich nur die Ganzjährigen abrechne, also das Stichdatum entsprechend setze. Für die anderen erstelle ich einen Zusatzbetrag: Mitglieder entsprechend filtern und allen den Beitrag zuweisen.
